### PR TITLE
usi-dock: Adjust the prompt content according to the customer's requi…

### PR DESCRIPTION
**Pull request by victor-cheng**
…rements.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: plugins/usi-dock/fu-usi-dock-mcu-device.c [hunk 1]
```diff
@@ -712,9 +712,9 @@ fu_usi_dock_mcu_device_insert_cb(gpointer user_data)
 	fwupd_request_set_kind(request, FWUPD_REQUEST_KIND_IMMEDIATE);
 	fwupd_request_set_id(request, FWUPD_REQUEST_ID_INSERT_USB_CABLE);
 	fwupd_request_add_flag(request, FWUPD_REQUEST_FLAG_ALLOW_GENERIC_MESSAGE);
-	fwupd_request_set_message(
-	    request,
-	    "The update will continue when the device USB cable has been re-inserted.");
+	fwupd_request_set_message(request,
+				  "Please insert the USB cable, then, new firmware will take "
+				  "effect after dock LED blinking finish.");
```
Comment (superm1) [inline plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: finish**es**, no?

Comment (victor-cheng) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: Yes, after the LED stops flashing, the upgrade will be completed and take effect. Then the host can be able to detect the insertion.

Comment (superm1) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: Yeah I was just making a suggestion in your wording.

Comment (victor-cheng) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: Thanks, these statements are directly defined and approved by the customer.

Comment (hughsie) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: But the text doesn't make sense.

Comment (hughsie) [reply plugins/usi-dock/fu-usi-dock-mcu-device.c:717]: What about we have `Please insert the USB cable to allow the update to continue.`

**Comment by hughsie** [comment]
Obsoleted by https://github.com/fwupd/fwupd/pull/6122

**Pull request closed without merging**